fractal-puppy/Engine
# Miscellaneous Bug resolving

# Bugs solved
- Fix levelup audio
- Fix new item/level feedback 
- Fix flames position

# Test
- Check that games no longer crashes on level up
- Check that when picking a new item, a cool exclamation icon appears near the inventory icon
- Check that flames are not floating anymore
